script/hospital.js

```diff
@@ -0,0 +1,38 @@
+class Hospital {
```

Comment:
You must add this file to index.html

Reply:
Oops, sorry about the logic typo, all fixed.

Reply:
Ok, no problem :)